src/components/number-picker/number-picker.component.ts

```diff
@@ -10,6 +10,9 @@ export const NUMBER_PICKER_VALUE_ACCESSOR: any = {
     multi: true
 };
 
+export const LARGE_INT_PRECISION: number =  Number.MAX_SAFE_INTEGER.toString().length - 1;
+export const DEFAULT_PRECISION: number = 6;
```

Comment:
I don't think either of these should be exported.

Reply:
@donna-marijne. I've moved these tests to Karma, I just thought testing the interaction may be difficult in karma tests but I was wrong. I'm thinking it could be a worthwhile exercise to move all the increment/decrement tests into the karma tests at the same time to speed up e2e running. What do you think?